docs/README.md

```diff
@@ -1,29 +1,22 @@
 # Unofficial Talon Docs
 
-Documenting my learning experiences with Talon, the speech-to-text system. These docs are not authorative or complete, but instead represent an 80/20 view of the API sufficient for getting started making extensions and using the system.
+Documenting my learning experiences with Talon, the speech-to-text system. These docs are not authorative or complete. After reading them you should be able to extend and use Talon.
 
 
 ## Intro
 
-Talon is a next-generation speech-to-text system for coding, commanding, and dictating to a computer. It is currently in open beta and is under active development by creator, Ryan Hileman, as of April 2018.
+Talon is a speech-to-text system for commanding and dictating to a computer, aimed at helping programmers write code without a keyboard. It is currently in open beta and is under active development by creator Ryan Hileman as of March 2019.
 
-Talon provides very good recognition and a still-developing set of APIs for extending it and adding custom phrases. Current versions require the use of [Dragon Naturally Speaking](https://www.nuance.com/dragon.html) and macOS at this time. The Talon project also includes support for the [Tobii 4C](http://a.co/bILYudx) eye tracker, which can allow the replacement of a mouse.
+You can extend Talon, for example add custom phrases. MacOS ≥ 10.11 is required; Linux is not yet supported. You can use ($150) [Dragon Naturally Speaking](https://www.nuance.com/dragon.html) or Apple’s (free) default text-to-speech engine. The Talon project also supports the [Tobii 4C](http://a.co/bILYudx) eye tracker, which can replace the mouse.
```

Comment:
Again, avoid the "you" language.

Don't list what isn't supported (Linux). List what is supported.

Don't list prices of software that can change at any time ($150).